services/appwrite.ts

```diff
@@ -1,4 +1,14 @@
-import { Appwrite, Models, Query } from "appwrite";
+import { Account, Avatars, Client, Databases, Storage, Models, Query } from "appwrite";
+
+const client = new Client()
+
+const account = new Account(client)
+
+const databases = new Databases(client);
```

Comment:
Please check lines for databases, in v1.0 they must specify databaseId like `almost-netflix-project-db`

```diff
-await databases.listDocuments<AppwriteMovie>("movies", "trendingIndex", ["DESC"]);
+await databases.listDocuments<AppwriteMovie>("almost-netflix-project-db","movies", "trendingIndex", ["DESC"]);
```

I have updated the seed project so you have a good working example of this 

https://github.com/Meldiron/almost-netflix-project-setup/blob/cacc3974a625411d97c4efc5a1e33c8228a3f08f/src/migrations.js#L125